### PR TITLE
Spanish Translation

### PR DESCRIPTION
**Pull request by KeeJef**
This will need to be checked for accuracy, PRing it for review

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import enUS from "./en-us"
 const languages = [
     { name: "English", code: "en-us", flag: "us" },
     { name: "Russian", code: "ru", flag: "ru" }
+	{ name: "Spanish", code: "es", flag: "es" }
```
Comment (Mikunj) [inline src/i18n/index.js]: There needs to be a comma after `{ name: "Russian", code: "ru", flag: "ru" }`. Make sure it's aligned properly too

File: src/plugins/timeago.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ export default ({
         locale: "en",
         locales: {
             ru: require("date-fns/locale/ru")
+			es: require("date-fns/locale/es")
```
Comment (Mikunj) [inline src/plugins/timeago.js]: same as above.

**Comment by KeeJef** [comment]
Fixed, still waiting on language review

File: src/i18n/es-es.js [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+        addressBookDetails: "Detalles de la libreta de direcciones",
+        addressBookIsEmpty: "La libreta de direcciones está vacía",
+        addresses: {
+            myPrimaryAddress: "Mi dirección principal",
+            myUnusedAddresses: "Mis direcciones no usadas",
+            myUsedAddresses: "Mis direcciones usadas",
+            notYourAddress: "¡No es una dirección que le pertenezca!",
+            primaryAddress: "Dirección principal",
+            subAddress: "Dirección auxiliar",
+            subAddressIndex: "Índice",
```
Comment (Mikunj) [inline src/i18n/es-es.js]: `subAddressIndex` needs to be updated to match #49.

**Comment by Mikunj** [comment]
We also might need to add/remove strings from #54. I've listed the translations i have removed and the ones i have added. 

Ref to [this](https://github.com/loki-project/loki-electron-gui-wallet/pull/54/files#diff-ef2ef6aa88150721cff32a793a3b10a0) for the diff.

**Comment by KeeJef** [comment]
Okay is this Good to merge @Mikunj ?

**Comment by Mikunj** [comment]
Good to go 👍

**Pull request merged**